server/_ide_helper.php111

```diff
@@ -3,7 +3,7 @@
 
```

Comment:
what is this file for?

Reply:
This file should help to avoid warnings shown by IDE(phpstorm). Should not be visible in git repo. I will move it to gitignore.